core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -50,6 +52,7 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
   public customColumnWidth: Record<string, string> = {
     "": "70px",
     Name: "230px",
+    "Workflow Version Sample": "190px",
```

Comment:
can you increase the column size so that the text doesn't wrap to the next line

Reply:
done